src/ocean/task/util/Timer.d

```diff
@@ -83,7 +80,27 @@ unittest
 {
     initScheduler(SchedulerConfiguration.init);
 
-    .timer = new typeof(timer);
+    class SimpleTask : Task
```

Comment:
Is this related to "Create global timer lazily inside helper"?

Reply:
Yes, diff is confusing. I have split unittest that requires resetting timer instance manually to check allocated slot count from documented unittest which uses regular lazily created timer.